corehq/apps/userreports/views.py

```diff
@@ -1283,7 +1283,7 @@ def export_data_source(request, domain, config_id):
 
     # build export
     def get_table(q):
-        yield list(table.columns)
+        yield list(table.columns.keys())
```

Comment:
@millerdev @nickpell this wasn't a dict fyi

was `<sqlalchemy.sql.base.ImmutableColumnCollection object at 0x10edd5d70>`